alphadia/constants/default.yaml

```diff
@@ -189,8 +189,23 @@ search_output:
   # can be either "parquet" or "tsv"
   file_format: "tsv"
 
+optimization:
+  order_of_optimization: null
+  ms2_error:
+      update_interval: 0.95
```

Comment:
sure these are all correct? would have expected 0.99 and 1.1 here for some ..

Reply:
It depends on whether we want to allow distinct update rules for targeted and automatic calibration. I decided not to (since any given run will only use one of these) and, since the emphasis so far has been on preventing any performance degradation for the targeted case, I thought it best to use the default values for this case in the YAML.

Reply:
I assume this is experimental until we find something which always work?

I agree with Magnus, let's use different parameters, especially as they wont have the same metric update rules in the future. If we dont want to expose this in the gui, make it accessible, we don't ned to worry about having to many parameters.